docs/java/java-project.md

```diff
@@ -9,49 +9,43 @@ MetaDescription: Lightweight Mode, Maven Support, Java Package and Dependency Ma
 MetaSocialImage:
 ---
 
-# Java project management in VS Code
+# Manage Java Projects in VS Code
 
-This document will give you an overview of how to manage your Java project in Visual Studio Code.
-
-If you run into any problems when using the features below, you can contact us by entering an [issue](https://github.com/microsoft/vscode-java-pack/issues).
-
-## Project management
-
-Managing a project in VS Code requires the [Project Manager for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-dependency) extension. The extension helps manage class paths and dependencies, and create new projects, packages, and classes. The Project Manager for Java extension is included in the [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack), which also has other tools for Java development in VS Code.
+The [Project Manager for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-dependency) extension helps you to manage your Java projects and their dependencies. It also helps you to create new Java projects, packages, and classes. To get the complete Java language support in Visual Studio Code, you can install the [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack), which includes the Project Manager for Java extension.
 
 <a class="tutorial-install-extension-btn" href="vscode:extension/vscjava.vscode-java-pack">Install the Extension Pack for Java</a>
 
-For details on how to get started with the extension pack, please see [Getting Started](/docs/java/java-tutorial.md).
+> For details on how to get started with the extension pack, please see [Getting Started](/docs/java/java-tutorial.md).
 
 ### Project view
 
-Project view helps you view your project and its dependencies, and provides entry points for project management tasks. You can switch between a hierarchy view and flat view.
+The project view helps you view your Java projects and their dependencies, and provides entry points for project management tasks. 
```

Comment:
Should we add an intro to where project view is? something like this:

The Java project view is displayed in the explorer view and you can toggle its display by clicking "..."  on the explorer area and selecting "Java Project"

Reply:
How about:

By default, the Java project view is displayed below the explorer view. If you cannot see it. Try clicking the `...` button next to the `EXPLORER` and select `Java Projects`.

Reply:
sounds good!